doc/contributor/parser.md

```diff
@@ -4,13 +4,15 @@
 
```

Comment:
Note that the entire `Setup` section here is automated in jt, maybe we should just get rid of it.